ArduPlane/is_flying.cpp

```diff
@@ -155,6 +155,9 @@ void Plane::update_is_flying_5Hz(void)
     }
     previous_is_flying = new_is_flying;
     adsb.set_is_flying(new_is_flying);
+#if FRSKY_TELEM_ENABLED == ENABLED
+    frsky_telemetry.set_is_flying(new_is_flying);
```

Comment:
I see you copied the ADSB example on the line above, sensible thing to do. However, that method is wrong and needs to be fixed. It should use the probability and read as:
frsky_telemetry.set_is_flying(is_flying());